src/codeCache.h

```diff
@@ -100,13 +100,38 @@ class CodeBlob {
 
 class FrameDesc;
 
+class CodeCache;
+
+class UnloadProtection {
+  private:
+    CodeCache* _protected_cc;
+    void* _lib_handle;
+
+  public:
+    UnloadProtection(CodeCache *cc);
+    UnloadProtection(UnloadProtection&& other) {
+        _protected_cc = other._protected_cc;
+        _lib_handle = other._lib_handle;
+        other._lib_handle = nullptr;
+    }
```

Comment:
This boilerplate is not needed anymore.

Reply:
79da179bcf966e9afcf92643f0c04a59946f6749